docs/generate-versions.py

```diff
@@ -0,0 +1,6 @@
+import os
```

Comment:
@andygrove please help to understand if this is a temporary helper file? If it is a perm file we should be describing the usage of it?

Reply:
Thanks. I added some comments